task/04-arrays-tasks.js

```diff
@@ -353,7 +404,7 @@ function getItemsSum(arr) {
  *  [ null, undefined, NaN, false, 0, '' ]  => 6
  */
 function getFalsyValuesCount(arr) {
-   throw new Error('Not implemented');
+   return arr.reduce((element1, element2) => element1 + (Boolean(element2) === false), 0);
```

Comment:
`return arr.reduce((element1, element2) => element1 + (!Boolean(element2)), 0);` this works
Please, see https://javascript.info/logical-operators